examples/medplum-chart-demo/data/example/example-patient-data.json

```diff
@@ -0,0 +1,833 @@
+{
+  "resourceType": "Bundle",
+  "type": "batch",
+  "entry": [
+    {
+      "fullUrl": "urn:uuid:badc91d9-9209-49b8-ab22-5bb44e75ae57",
+      "request": {
+        "method": "POST",
+        "url": "Patient"
+      },
+      "resource": {
+        "resourceType": "Patient",
+        "name": [
+          {
+            "given": ["Homer"],
+            "family": "Simpson"
+          }
+        ],
+        "gender": "male",
+        "maritalStatus": {
+          "coding": [
+            {
+              "system": "http://hl7.org/fhir/ValueSet/marital-status",
+              "code": "M",
+              "display": "Married"
+            }
+          ]
+        },
+        "birthDate": "1956-05-12"
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:60081479-c0a1-4b9e-9c52-039cf80f45b4",
+      "request": {
+        "method": "POST",
+        "url": "Observation"
+      },
+      "resource": {
+        "resourceType": "Observation",
+        "status": "final",
+        "subject": {
+          "reference": "urn:uuid:badc91d9-9209-49b8-ab22-5bb44e75ae57"
```

Comment:
Can you manually add the display strings for these references? Really helps when demoing